### PR TITLE
Test avro to json

### PR DESCRIPTION
**Pull request by Sunn-y-Arora**
## Description
A follow up PR of #204 ,  for changing the test using JSON.
changes:
 GSON library for json serialisation/de-serialisation

### Checklist:
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: hypertrace-view-generator/hypertrace-view-generator/build.gradle.kts [hunk 1]
```diff
@@ -39,6 +39,7 @@ dependencies {
 
   implementation("org.apache.avro:avro:1.10.2")
   implementation("org.apache.commons:commons-lang3:3.12.0")
+  implementation("com.google.code.gson:gson:2.8.7")
```
Comment (kotharironak) [inline hypertrace-view-generator/hypertrace-view-generator/build.gradle.kts:42]: It should be testImplementation, right?

**Comment by kotharironak** [comment]
Can you also check why the build is failing?

File: hypertrace-view-generator/hypertrace-view-generator/build.gradle.kts (hunk 1 above)
Comment (Sunn-y-Arora) [reply hypertrace-view-generator/hypertrace-view-generator/build.gradle.kts]: yes

**Comment by Sunn-y-Arora** [comment]
@kotharironak fixed the build, pls review

File: hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/utils/ByteBufferTypeAdapter.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package org.hypertrace.viewgenerator.generators.utils;
+
+import com.google.gson.*;
```
Comment (kotharironak) [inline hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/utils/ByteBufferTypeAdapter.java:3]:
```suggestion
import com.google.gson.JsonDeserializationContext;
import com.google.gson.JsonDeserializer;
import com.google.gson.JsonElement;
import com.google.gson.JsonPrimitive;
import com.google.gson.JsonSerializationContext;
import com.google.gson.JsonSerializer;
```

Comment (kotharironak) [reply hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/utils/ByteBufferTypeAdapter.java:3]: nit : Lets be explicit about package

**Review by kotharironak — COMMENTED** [review]
Lgtm, have one nit comment, can you accept it?

**Review by kotharironak — APPROVED** [review]

**Pull request merged**